Contuity/Models/FollowUp.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  FollowUp.swift
+//  Contuity
+//
+//  Created by Lexi Spikerman on 10/2/18.
+//  Copyright © 2018 Generate. All rights reserved.
+//
+
+import Foundation
+
+class FollowUp {
+    
+    var id: Int
+    var jot_id: Int
+    var text: String?
+    var datetime: NSDate?
```

Comment:
Is there any specific reason we are choosing to go with NSDate over Date?

Also, nitpicky but
- id and jot id are meant to be immutable right? They should technically be lets
- jot_id => jotId ? can we decide on style tomorrow meeting